source/renderer/util/load-icons.js

```diff
@@ -14,24 +14,14 @@
 
 const clarityIcons = require('@clr/icons').ClarityIcons
 require('@clr/icons/shapes/all-shapes')
-const fs = require('fs').promises
-const path = require('path')
-const isDir = require('../../common/util/is-dir')
 
-async function loadCustomIcons (dir) {
-  let directoryContents = await fs.readdir(dir)
-  for (let file of directoryContents) {
-    let fullPath = path.join(dir, file)
-    if (isDir(fullPath)) return loadCustomIcons(fullPath)
-
-    let ext = path.extname(file)
-    if (ext && ext.toLowerCase() === '.svg') {
-      let content = await fs.readFile(fullPath, { encoding: 'utf8' })
-      let icon = {}
-      icon[path.basename(file, ext)] = content
-      clarityIcons.add(icon)
-    }
-  }
+async function loadCustomIcons () {
+  /* eslint-disable import/no-webpack-loader-syntax */
+  clarityIcons.add({
+    'code-alt.svg': require('!!svg-inline-loader!./../assets/icons/clarity-custom/code-alt.svg'),
```

Comment:
Love it!

But we should definitely add a comment to tell people why we are using `!!svg-inline-loader!` prepended to the path. This will make it easier for other people understand this beautiful (yet full-of-assumptions) code!